### PR TITLE
Add tensorboard logging + fix typos, not setting weights in init

### PR DESCRIPTION
**Pull request by Kasperskyy**

### CODE REVIEW

File: nst/models/baseline.py [hunk 1]
```diff
@@ -74,19 +76,31 @@ def __init__(
...
+        writer = SummaryWriter()
+        writer.add_image("Working Image", self._optimized_image.squeeze(), global_step=self.global_step)
+        writer.add_scalar("Loss/content/", content_loss, global_step=self.global_step)
+        writer.add_scalar("Loss/style/", style_loss, global_step=self.global_step)
+        writer.add_scalar("Loss/total_variation/", total_variation_loss, global_step=self.global_step)
+        writer.close()
+
+        # Open Tensorboard with the command:
+        # tensorboard --logdir=runs
+        # the runs folder is generated by default in the same path as the python script (here it would be nst/models)
```
Comment (Walusus) [inline nst/models/baseline.py:103]: I'd remove this comment or move it to README to keep the code clear :)

**Review by Walusus — CHANGES_REQUESTED** [review]
The code itself looks good :100:
I'd only remove or move the tenosrboard comment to README.

File: nst/models/baseline.py (hunk 1 above)
Comment (Kasperskyy) [reply nst/models/baseline.py:103]: That's fair enough :) I shall move it now

**Review by Walusus — APPROVED** [review]
It's ready to go now :+1:

**Pull request merged**